app/app.rb

```diff
@@ -407,15 +407,8 @@ def initialize
       elsif mongoid?
         account = Account.find_by(email: /^#{Regexp.escape(params[:email])}$/i)
       end
-      if account
-        if account.reset_password!
-          flash[:notice] = "A new password was sent to #{account.email}"
-        else
-          flash[:error] = "There was a problem resetting your password."
-        end
-      else
-        flash[:error] = "There's no account registered under that email address."
-      end
+      account.reset_password!
```

Comment:
Will this be okay if `account` isn't found, or will it be calling a method on nil? Might need `if account.present?` on the end?

Reply:
Maybe add a quick request spec for 'attempting to login with an unknown email address'.